src/main/resources/application.conf

```diff
@@ -198,26 +198,49 @@ mantis {
     transaction-timeout = 2.minutes
   }
 
-  mining {
-    # Maximum number of ommers kept in the pool
-    ommers-pool-size = 30
+  consensus {
+    # FIXME this was moved here from the `mining` section
+    active-timeout = 5 seconds
 
-    # This determines how many parallel eth_getWork request we can handle, by storing the prepared blocks in a cache,
-    # until a corresponding eth_submitWork request is received
-    block-cashe-size = 30
+    # New timeout to replace `ommer-pool-query-timeout` for uses outside ethash mining
+    get-transaction-from-pool-timeout = 5.seconds
 
+    # Moved from the `mining` section
```

Comment:
Those comments serve as documentation, not a changelog. Do you plan on keeping the comments with historical info or are they just temporary?

Reply:
Just temporary, when the final places are decided these comments will be removed. We do not even have a `mining` section anymore, it is called `ethash` now, following up on a comment by @LukasGasior1 in #395